src/ivfs/ivfs_tree.ml

```diff
@@ -267,8 +254,7 @@ module Make (Store: STORE) = struct
     let diff x y =
       let set t =
         let acc = ref KVSet.empty in
-        iter t (fun k v ->
-            v () >>= fun (v, _) ->
+        iter t (fun k (v, _) ->
```

Comment:
Do we care about changes to file types here too (normal/link/exec)?
(this PR doesn't change anything; just a comment)